### ISSUE TITLE
Q: how important is selectorConcat?

### ISSUE THREAD

**Issue by giuseppeg** [SlexAxton/rework-pseudo-classes#7]
Hey @SlexAxton is [`selectorConcat`](https://github.com/SlexAxton/rework-pseudo-classes/blob/master/lib/index.js#L127) really necessary? 
`:focus.pseudo-hover` works anyway, why do you need to invert the pseudo classes?

**Comment by SlexAxton** [comment]
I spose it's probably not necessary if that works everywhere, in my mind it felt weird

**Comment by giuseppeg** [comment]
Agreed that it feels weird but it should work everywhere unless the w3 validator is a liar.

I am asking because I am adding a [_generate-only-these-pseudo-classes_ feature](https://github.com/giuseppeg/postcss-pseudo-classes/pull/3) to the postcss version and `allCombinations` doesn't play nicely with the new option if the classes are inverted.

Btw would you be my reviewer?

**Comment by SlexAxton** [comment]
I'd be a little worried about other 3rd party tools that don't expect this. Can you just fix it after you run `allCombinations`?

Happy to review.

**Comment by giuseppeg** [comment]
<blockquote>Can you just fix it after you run allCombinations?</blockquote>

I wanted to avoid the overhead and to hack on top of the existing code, but yeah ideally I could.
